### ISSUE TITLE
Seeing errors with socket.io connections from ev-dashboard

### ISSUE THREAD

**Issue by Pulse-akhil** [sap-labs-france/ev-dashboard#2583]
I have frontend'd the ev-dashboard with an nginx server acting as a reverse proxy. And I have also enabled the nginx proxy_set_headers for /socket.io/ path within the nginx conf file. Despite these the WS:// connections from the angular dashboard errors out.

here is the URL that the dashboard pings ws://cs.foo.com/socket.io/?auth_token=xxxxxx=4&transport=websocket 

Here is the error from the browser console 
2021-09-25T00:29:44.235Z :: SocketIO client connect error: Error: websocket error 

Any idea why this may be happening?

**Comment by LucasBrazi06** [comment]
Do you use Chrome?
Do you have an error at backend side?
The WebSocket should try to connect to the REST server with SocketIO enabled.
FYI, I planned to replace SocketIO by uSocket because of scaling issues with IO.

**Comment by Pulse-akhil** [comment]
Yes, using Chrome for testing. And yes they are going to the REST central system service. 

I haven't noticed errors in the backend. Do you happen to know if these error logs will be in the DB? 

What's the functionality of the websocket connection? 

Also, if there is documentation or manual on how to hook up a charging station to the server that will be really helpful for new folks.

**Comment by LucasBrazi06** [comment]
SocketIO is used to refresh the front-end when data is changed in the DB through the REST server.
So you will not have refresh of lists on DB change in that case.
I'll check and drop you a doc to connect a charging station.

**Comment by LucasBrazi06** [comment]
All docs I found are internal, I requested an external doc but not before next week (we'll enhance the readme.)
In the meantime I can explain.
Once you have created an organization and logged into it, you'll have a charging station menu (1) where there is a tab named 'Onboard Charging Station' (2).
In this tab, you can create a token (3) with an expiration date and copy/paste the connection URL according the protocol of your charging station (4).

<img width="977" alt="2021-09-25_11-01-26" src="https://user-images.githubusercontent.com/25266945/134765987-fe3d5b22-1c3d-4894-97b5-22516b9645c6.png">

Then reboot your charging station and you should see it in the dashboard.

<img width="984" alt="2021-09-25_11-08-52" src="https://user-images.githubusercontent.com/25266945/134766167-9fdd2e8e-4055-4285-a867-6fd9c183129b.png">

**Comment by Pulse-akhil** [comment]
I am setting up the ev-simulator to test this out. Fingers crossed it works.
